lib/nuget-provider.js

```diff
@@ -137,7 +138,14 @@ function mapNugetFeed({ camp, cache }, pattern, offset, getInfo) {
       intervalMillis: 42 * 60 * 1000,
       scraper: json => json.resources.filter(resource => resource['@type'] === 'SearchQueryService'),
     }, (err, searchQueryResources) => {
-        if (err != null) { done(err); return; }
+        if (err != null) {
+          if (err.stack.indexOf("JSON.parse") !== -1) {
```

Comment:
Good shout. It certainly was brittle - it turns out the reason some of the tests failed in CI but not on my machine was because the stacktrace here was different in Node 6 and Node 8. Have updated accordingly :)